lms/djangoapps/teams/static/teams/js/spec/collections/topic_collection_spec.js

```diff
@@ -69,5 +69,11 @@ define(['URI', 'underscore', 'common/js/spec_helpers/ajax_helpers', 'teams/js/co
                 topicCollection.course_id = 'my+course+id';
                 testRequestParam(this, 'course_id', 'my+course+id');
             });
+
+            it ('refreshes the page on "teamModel:add" event', function () {
+                spyOn(topicCollection, 'goTo');
+                Backbone.trigger('teamModel:add', {});
+                expect(topicCollection.goTo.calls[0].args[0]).toBe(1);
```

Comment:
I don't think that this test belongs on the collection. There should be tests on views that verify they updated when a new team is added to the collection. I'd also recommend that you actually add a model to the collection, rather than faking the event.